content/2_Bronze/Intro_Sets.mdx

```diff
@@ -17,109 +19,156 @@ prerequisites:
 	</Resource>
 </Resources>
 
-<LanguageSection>
-<CPPSection>
+A **set** is a collection of unique elements. Sets have three primary methods:
 
-C++ contains two versions of sets and maps; one using **sorting** and the other
-using **hashing**.
+ - one to add an element
+ - one to remove an element
+ - one to check whether an element is present
 
-</CPPSection>
-<JavaSection>
 
-Java contains two versions of sets and maps; one using **sorting** and the other
-using **hashing**.
-
-</JavaSection>
-<PySection>
+A **map** is a collection of entries, each consisting of a **key** and a **value**. In
```

Comment:
maybe clarify that a map = dictionary in python

Reply:
it is mentioned later